### ISSUE TITLE
aria-disabled on nav buttons is set to early when using slider asNavFor other slider

### ISSUE THREAD

**Issue by herrschuessler** [kenwheeler/slick#2343]
###### `[  http://codepen.io/herrschuessler/pen/YWzOYO  ]`  
Button will turn red when aria-disabled

====================================================================

#### Steps to reproduce the problem
1. We have two sliders. Slider A is showing one slide at a time. Slider B is showing n items at a time but sliding one item at a time. Slider B acts `asNavFor` slider A.
2. Navigate via next button

====================================================================

#### What is the expected behaviour?

Next button should be aria-disabled only once last slide in slider B is reached

====================================================================

#### What is observed behaviour?

Next button is aria-disabled once last *set of slides* in slider B is reached

**Comment by leggomuhgreggo** [comment]
There are a few aria issues, [here's the tracking one](https://github.com/kenwheeler/slick/issues/1820), and a [PR pending](https://github.com/kenwheeler/slick/pull/2119) to address at least some of the issues.

It'd be great if you could check to see whether that PR solves this issue. Thanks.

**Comment by JoshuaKGoldberg** [comment]
Is this still an issue? #2119 has been merged.
